gnucash/gnome/gnc-budget-view.c

```diff
@@ -987,15 +988,7 @@ budget_col_source(Account *account, GtkTreeViewColumn *col,
     }
 
     note = gnc_budget_get_account_period_note(budget, account, period_num);
-
-    if (note == NULL)
-    {
-        g_object_set(cell, "background", NULL, NULL);
-    }
-    else
-    {
-        g_object_set(cell, "background", "beige", NULL);
-    }
+    g_object_set(cell, "flagged", note != NULL, NULL);
```

Comment:
As the flagged text entry widget replaces the background color changes from an earlier commit, consider squashing them together.

Reply:
OK, added a css styling.
Changed to "blue" so it is visible also on rows highlighted with selection.

Reply:
Unfortunately on my system the highlight color is also blue, so the flag corner disappears in that context.

Reply:
Switched to using "selected" and "normal" background colors for flag. No custom css